resilience4j-spring/src/main/java/io/github/resilience4j/circuitbreaker/configure/CircuitBreakerConfigurationProperties.java

```diff
@@ -45,12 +48,28 @@ public void setCircuitBreakerAspectOrder(int circuitBreakerAspectOrder) {
     }
 
     private BackendProperties getBackendProperties(String backend) {
-        return backends.get(backend);
+    	BackendProperties properties = backends.get(backend);
+    	if(properties.getSharedConfigName() != null && !properties.getSharedConfigName().isEmpty() ) {
```

Comment:
can we replace it with  if(StringUtils.isEmpty(backendProperties.getSharedConfigName())) { for better code readability